src/compounds/ActionsGroup/ActionsGroup.jsx

```diff
@@ -24,12 +23,10 @@ const ActionsGroup = ({ initialFiles }) => {
   return (
     <>
       <ListGroup className='actions-group'>
-        {/* TODO(@summercook): return this and the below commented code
-        once we are able to refactor posting messages & attachments */}
-        {/* <ListGroup.Item action variant='primary' onClick={() => handleShow('SendMessage')}>
+        <ListGroup.Item action variant='primary' onClick={() => handleShow('SendMessage')}>
```

Comment:
```suggestion
        <ListGroup.Item
          action
          variant='primary'
          onClick={() => handleShow('SendMessage')}
        >
```

Reply:
to match the formatting of the next item.

Reply:
also, I think the variants are supposed to alternate? so either this one or the next will need to change.

Reply:
on second thought...I had to play around with the item below in the full color demo so you can leave this variant and I'll edit it on my branch.